Marlin/src/lcd/dwin/e3v2/dwin.cpp

```diff
@@ -4564,6 +4564,10 @@ void octo_make_name_without_ext(char *dst, char *src, size_t maxlen = MENU_CHAR_
 void HMI_SDCardInit() { card.cdroot(); }
 
 void MarlinUI::refresh() { /* Nothing to see here */ }
+void MarlinUI::set_remaining_time (const uint32_t r) { _remain_time = r; }
+void MarlinUI::set_progress(const progress_t p) { _card_percent = _MIN(p, 100U * (PROGRESS_SCALE)); }
```

Comment:
I don't really understand what this means and how to properly use it. I would really love to use decimals for better progress estimation, which means `_card_percent` should be always treated as something to be multiplied by `PROGRESS_SCALE`, but I need to delve deeper into how to apply it properly. This is why this PR is in draft state so far.